tests/src/test/scala/com/evolutiongaming/kafka/journal/IntegrationSuite.scala

```diff
@@ -14,52 +12,50 @@ import com.evolutiongaming.scassandra.CassandraClusterOf
 import com.evolutiongaming.smetrics.CollectorRegistry
 import com.github.dockerjava.api.model.{ExposedPort, HostConfig, PortBinding, Ports}
 import com.typesafe.config.ConfigFactory
+import org.testcontainers.cassandra.CassandraContainer
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.kafka.KafkaContainer
+import org.testcontainers.utility.DockerImageName
 
 object IntegrationSuite {
+  private val CassandraDockerImage = DockerImageName.parse("cassandra:4.1.8") // latest 4.x
+  private val KafkaDockerImage     = DockerImageName.parse("apache/kafka-native:3.8.1") // 3.9.0 doesn't start here
```

Comment:
can we make the version constants somehow exposed for Scala Steward's update suggestions? 

not likely, I guess, but we have to be able somehow to not lose them over time

Reply:
probably only by turning those constants into separate Scala library artefacts, with versions matching container versions. if you think it is worth it, please create a separate issue for it. I'm not sure it is worth it though.